src/codegen/properties/function_parameters.py

```diff
@@ -0,0 +1,93 @@
+"""Structure for storing function parameters metadata from scrapigen."""
+from codegen.properties.cluster_elements import ClusterElements
+
+
+class FunctionsParameter:
+    """Structure for storing function metadata from scrapigen."""
+
+    def __init__(self, parameters_metadata):
```

Comment:
`parameters_metadata` describes a single parameter so rename it to `parameter_metadata`.